libmscore/timesig.h

```diff
@@ -35,9 +36,26 @@ enum class TimeSigType : char {
 
 //---------------------------------------------------------------------------------------
 //   @@ TimeSig
-///    This class represents a time signature.
+//   This class represents a time signature.
```

Comment:
I think the `///`is needed for the plugin documentation?

Reply:
That is a fair cop but society is to blame.

Reply:
Sorry, I don't get that, is it an idiom that just doesn't translate for me?

Reply:
A Monty Python quote... :D

Reply:
Ah, and poor me saw them dubbed in German ;-)

Reply:
I can see how it might be confusing in German... :P